recipes/lb-nightly-rpc/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "lb-nightly-rpc" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lb-nightly-rpc-{{ version }}.tar.gz
+  sha256: 8c331c51c969b4fc7c18b85b8f4fc0c9b97cb56eea78963986cd14bc4cb93c7c
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
```

If the package is noarch it doesn't matter if the windows CI fails with missing dependencies:
```suggestion
  noarch: python
```

---

Ok,  thanks ;-)